daml-lf/scenario-interpreter/src/main/scala/com/digitalasset/daml/lf/Pretty.scala

```diff
@@ -80,6 +80,11 @@ private[lf] object Pretty {
 
       case Error.CanceledByRequest() =>
         text("Evaluation was cancelled because the test was changed and rerun in a new thread.")
+
+      case Error.NoSuchTemplate(templateId, packageId) =>
+        text(
+          s"Error: Could not find template $templateId from package $packageId to use in create/exercise."
```

Comment:
Consider other stringification methods for both, perhaps adding package metadata to the error to show the name+version